### PR TITLE
Add slow query list page e2e test

### PR DESCRIPTION
**Pull request by YiniXu9506**
- Add E2E tests for slow query list page
- Waiting for PR [fix: missing fields in slow query](https://github.com/pingcap/tidb-dashboard/pull/1150) to be merged

### CODE REVIEW

File: ui/cypress/integration/slow_query/list.spec.js [hunk 1]
```diff
@@ -236,6 +241,363 @@ describe('SlowQuery list page', () => {
...
+        let queryData = {
+          query: 'SELECT sleep(1.5);',
+          database: '',
+        }
+        cy.task('queryDB', { ...queryData })
+
+        cy.wait(2000)
+        cy.reload()
+        // global and use database queries will be listed
+        cy.get('[data-automation-key=query]').should('has.length', 3)
```
Comment (shhdgit) [inline ui/cypress/integration/slow_query/list.spec.js:255]: What is `[data-automation-key=query]`? I only find `[data-automation-key=name]` in html structs.

Comment (YiniXu9506) [reply ui/cypress/integration/slow_query/list.spec.js:255]: Each table column has a `data-automation-key` generated with `visibleColumnKeys` in CardTable.

image

File: ui/cypress/integration/slow_query/list.spec.js [hunk 2]
```diff
@@ -236,6 +226,364 @@ describe('SlowQuery list page', () => {
...
+          .then(() => {
+            cy.get('[data-automation-key=query]').should('has.length', 3)
+          })
+      })
+
+      it('Type search without pressing enter then reload', () => {
+        cy.get('[data-e2e=slow_query_search]').type(' SELECT sleep\\(1\\)')
+
+        cy.reload()
+        cy.get('[data-automation-key=query]').should('has.length', 3)
```
Comment (shhdgit) [inline ui/cypress/integration/slow_query/list.spec.js:280]: Maybe we want to verify that `[data-e2e=slow_query_search]` is empty?

File: ui/cypress/integration/slow_query/list.spec.js [hunk 3]
```diff
@@ -236,6 +226,364 @@ describe('SlowQuery list page', () => {
...
+
+      it('Check config remembered', () => {
+        cy.get('[data-e2e=slow_query_limit_select]')
+          .click()
+          .then(() => {
+            cy.get('[data-e2e=slow_query_limit_option]')
+              .eq(1)
+              .click()
+              .then(() => {
+                cy.get('[data-automation-key=query]').should('has.length', 3)
```
Comment (shhdgit) [inline ui/cypress/integration/slow_query/list.spec.js:311]: Why does this means that the limit config is remembered?

**Review by shhdgit — APPROVED** [review]
Learned a lot from this pr, the rest LGTM.

File: ui/cypress/integration/slow_query/list.spec.js (hunk 2 above)
Comment (YiniXu9506) [reply ui/cypress/integration/slow_query/list.spec.js:280]: In the previous `it` set, I search (with enter action) `SELECT sleep(1)` and get one result, then clear the search (with enter action) text and get 3 results. 

And In this `it` set, I want to test search without pressing enter. If we type text on search box without pressing enter, then the `/slow_query/list` will not be triggered, and the length of result should still be 3.

**Pull request merged**